src/libaktualizr-c/test/api-test-utils/api-test-utils.cc

```diff
@@ -0,0 +1,47 @@
+#include "api-test-utils.h"
+
+#include <boost/process.hpp>
+#include "../tests/httpfake.h"
+#include "../tests/uptane_test_common.h"
+#include "config/config.h"
+
+void Run_fake_http_server(const char *path) {
+  std::string port = TestUtils::getFreePort();
+  std::string server = "http://127.0.0.1:" + port;
+  Config config;
+  config.uptane.repo_server = server;
+  boost::process::child http_server_process(path, port, "-f");
+  TestUtils::waitForServer(server + "/");
+}
+
+UptaneGenerator *Get_uptane_generator(const char *path) { return new Process(path); }
+
+void Run_uptane_generator(UptaneGenerator *generator, const char **args, size_t args_count) {
+  std::vector<std::string> args_vector;
+  args_vector.reserve(args_count);
+  for (size_t i = 0; i < args_count; ++i) {
+    args_vector.emplace_back(args[i]);
```

Comment:
It's getting an hard-to-avoid lint warning:

```suggestion
    // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic)
    args_vector.emplace_back(args[i]);
```

Reply:
Outdated.
Anyway thanks for the explanation.